### DISCUSSION TITLE
Problem with actualizing the feeds

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#5534] [Q&A]
I have setup automatic feed actualization with a cronjob every 15 minutes, and ttl is set to 1 hour for all feeds. But in the application, I observe that feeds don't get updated for as long as 10 hours.

For instance, a TechCrunch article was published at 8:36 GMT, but it was not available in the main stream until around 18:00 GMT. Assuming that articles don't get added immediately to the RSS feed and taking a cut-off time of up to 1 hour, they should have been added to the database within 3 hours of the article being published.

Is there a maximum limit to the number of feeds that are actualized every time? And if yes, how can I increase the limit?

**Comment by Frenzie** [comment]
There's a bit of an edge case issue where an error can be cached, I forget the exact details. Maybe something like that happened? Or do you mean you've seen this multiple days in a row?

**Reply by ghost** [reply]
It happens most of the time on almost all the feeds. The other times, they do appear in the database within an hour or so. My subscriptions contain around 100 feeds, and I am wondering if the code or the server is restricting the number of feeds that refresh every time.
